libs/device/startup.ts

```diff
@@ -2,4 +2,11 @@
 
 game.setWaitAnyButton(controller.pauseUntilAnyButtonIsPressed)
 // force first game engine
-game.eventContext()
\ No newline at end of file
+game.eventContext()
+
+game.eventContext().registerFrameHandler(199, () => {
+    if (multiplayer.getOrigin() === "client") {
```

Comment:
@jwunderl This might be problematic. Could this be done in the online-multiplayer package?

Reply:
yes, it should be moved to online-multiplayer package -- this was strictly for ease of use / testing and the way we had it set up made it easier to leave it here at the time.